pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -288,7 +297,9 @@ func (l *LibvirtDomainManager) setGuestTime(vmi *v1.VirtualMachineInstance) erro
 
```

Comment:
@xpivarc did you try simply checking if `dom.Status.Status == api.Running` ?

Reply:
@vladikr 
I did not because I don't think it would help (well it would but not with the flood). The check if the domain is running only guards us before too early executions but it will not prevent us to flood the logs in case the agent is not there in the first place. Does it make sense?

Reply:
@xpivarc Well, if the domain is running and the agent is not there, this thread should just quit.